<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultWindow</class>
 <widget class="QDialog" name="resultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Unicode MS</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_note">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入您的名字</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>您的得分：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
